Makefile

```diff
@@ -190,6 +190,8 @@ else
 endif
 
 ifdef ASSERT_STATUS_CHECKED
+# For ASC, turn off constructor elision
```

Comment:
This "what" comment doesn't tell me anything I can't reverse engineer from about 3 seconds of reading the Makefile. Tell me *why*.

I can't even turn it off and see what breaks, because it only works to prevent regressions.